src/components/TwoLevelSelector.js

```diff
@@ -0,0 +1,377 @@
+const React = require('react');
+const ReactDOM = require('react-dom');
+const Radium = require('radium');
+
+const Icon = require('./Icon');
+const StyleConstants = require('../constants/Style');
+
+const TwoLevelSelector = React.createClass({
+
+  propTypes: {
+    data: React.PropTypes.object,
+    onChange: React.PropTypes.func,
+    selectedItem: React.PropTypes.string,
+    title: React.PropTypes.string
+  },
+
+  getDefaultProps () {
+    return {
+      data: null,
```

Comment:
It seems like the data prop should be required for this component to work. If that's the case, then we shouldn't set a default prop.